pkg/sources/github/github.go

```diff
@@ -1331,60 +1292,29 @@ func (s *Source) processIssueComments(ctx context.Context, repoInfo repoInfo, ch
 	}
 
 	for {
-		issueComments, _, err := s.apiClient.Issues.ListComments(ctx, repoInfo.owner, repoInfo.name, allComments, issueOpts)
+		issueComments, _, err := s.apiClient.Issues.ListComments(ctx, info.owner, info.repo, allComments, issueOpts)
 		if s.handleRateLimit(err) {
-			continue
+			break
```

Comment:
This should be `continue`.

Reply:
If I'm reading the original refactor commit correctly, this is an appropriate change for a revert. Did I miss some consideration?

Reply:
The revert was correct. However, some of these changes are fixes for things that didn't work prior. I'm just pointing out things that need to be re-fixed.

E.g., using break instead of continue means it will never retry these API requests.